File: agc012a.pl
use strict;
use warnings;
use 5.010;

my $n = <>;
my $ans = 0;
my $flag = 1;
chomp $n;
my @arr = split / /, <>;
chomp $arr[-1];
@arr = sort{$a <=> $b} @arr;

for(my $i = $n; $i < $n *3; $i++) {
	if($flag) {
		$ans += $arr[$i];
		$flag = 0;
	} else {
		$flag = 1;
	}
}
say $ans;

exit;

File: agc012a.cpp
#include <bits/stdc++.h>
using namespace std;
int main() {
	int n;
	cin >> n;
	long long int ans = 0;
	vector <int> arr(3 * n);
	for(int i = 0;i < 3 * n; i++) {
		cin >> arr[i];
	}
	sort(arr.begin(), arr.end());

	for(int i = n; i < 3 * n; i += 2) {
		ans += arr[i];
	}
	cout << ans << endl;

	return 0;
}
